ros/src/computing/perception/detection/packages/cv_tracker/nodes/obj_reproj/obj_reproj.cpp

```diff
@@ -192,6 +200,32 @@ static visualization_msgs::MarkerArray convert_marker_array(const cv_tracker::ob
   return ret;
 }
 
+#ifdef IS_FOUND_JSK_PLUGIN
+static jsk_recognition_msgs::BoundingBoxArray convertJskBoundingBoxArray(const cv_tracker::obj_label& src)
+{
+  jsk_recognition_msgs::BoundingBoxArray ret;
+
+  for (const auto& reproj_pos : src.reprojected_pos)
+    {
+      jsk_recognition_msgs::BoundingBox bounding_box;
+      /* Set frame ID */
```

Comment:
I think comments in this block are redundant and unnecessary. We can understand it from fields name of class/struct.